src/core/datastore/datastore.js

```diff
@@ -80,6 +80,7 @@ export class DataStore {
    */
   static clearCache(options = {}) {
     return repositoryProvider.getOfflineRepository()
-      .then(repo => repo.clear());
+      .then(repo => repo.clear())
```

Comment:
What does `repo.clear()` return? Just curious.

Reply:
It doesn't need to return anything. Right now, I think it returns `true`. But a clear, in my mind, either succeeds, or fails (resolves a promise, or rejects it) - I can't think of a meaningful result for it.

In some later PR, I've set the result of `DataStore.clearCache()` to be `null`, for backwards compatibility.

Reply:
Ok. I agree that a clear either succeeds or fails. I just wanted to check to see if it was returning something meaningful that might be helpful. Returning `null` for backwards compatibility makes sense.